cmd/kind/create/cluster/createcluster.go

```diff
@@ -50,15 +52,25 @@ func NewCommand() *cobra.Command {
 	}
 	cmd.Flags().StringVar(&flags.Name, "name", "1", "cluster context name")
 	cmd.Flags().StringVar(&flags.Config, "config", "", "path to a kind config file")
+	cmd.Flags().Int32Var(&flags.ControlPlanes, "control-planes", 1, "number of control-plane nodes in the cluster")
```

Comment:
'control-planes' sounds confusing to me. It kind of suggests to me that there could be multiple clusters.

Perhaps this could be something more like `--control-plane-nodes`? Or `--num-nodes control-plane=3 --num-nodes workers=3`? 🤔

Reply:
the flags can be renamed `control-plane-nodes`, `worker-nodes`
or something like `--nodes=control-planes=3;workers=5`

but this reminds me of something else.
we also wanted to support multi-cluster with kind, so the flags have to target a cluster. o_o

Reply:
--name controls the target cluster for multi cluster. we already support that :-)